drlhp/pref_db.py

```diff
@@ -185,16 +184,15 @@ def get_dbs(self):
         return train_copy, val_copy
 
     def wait_until_len(self, min_len):
-        min_val_len = int(0.3*min_len)
         while True:
             self.lock.acquire()
             train_len = len(self.train_db)
             val_len = len(self.val_db)
             self.lock.release()
-            if train_len >= min_len: # and val_len > min_val_len:
+            if train_len >= min_len and val_len > 0:
                 break
             print("Waiting for preferences; {}/{} train so far, {}/{} val ".format(train_len,
                                                                                    min_len,
                                                                                    val_len,
-                                                                                   min_val_len))
+                                                                                   0))
```

Comment:
```suggestion
                                                                                   1))
```
Since you need 1 validation example